### PR TITLE
MusicPlayer2: Add version 2.73

### PR DESCRIPTION
**Pull request by hu3rror**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/musicplayer2.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        [
+            "MusicPlayer2.exe",
+            "MusicPlayer2"
+        ]
+    ],
+    "pre_install": [
+        "if(!(Test-Path \"$persist_dir\\recent_path.dat\")) { New-Item \"$dir\\recent_path.dat\" -ItemType File -Force | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\song_data.dat\")) { New-Item \"$dir\\song_data.dat\" -ItemType File -Force | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\config.ini\")) { New-Item \"$dir\\config.ini\" -ItemType File -Force | Out-Null }"
+    ],
```
Comment (rashil2000) [inline bucket/musicplayer2.json:28]:
```suggestion
    "pre_install": "'recent_path.dat', 'song_data.dat', 'config.ini' | ForEach-Item { if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType File -Force | Out-Null } }",
```

File: bucket/musicplayer2.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "persist":[
+        "playlist",
```
Comment (rashil2000) [inline bucket/musicplayer2.json:30]: `playlist` is a folder?

File: bucket/musicplayer2.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "recent_path.dat",
+        "song_data.dat",
+        "config.ini"
+    ],
+    "checkver": {
+        "github": "https://github.com/zhongyang219/MusicPlayer2"
+    },
```
Comment (rashil2000) [inline bucket/musicplayer2.json:37]:
```suggestion
    "checkver": "github",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/musicplayer2.json (hunk 2 above)
Comment (hu3rror) [reply bucket/musicplayer2.json:30]: yep, this is a folder

File: bucket/musicplayer2.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://github.com/zhongyang219/MusicPlayer2",
+    "description": "MusicPlayer2 is an audio player which with a collection of music playback, lyrics display, format conversion and many other functions.",
```
Comment (rashil2000) [inline bucket/musicplayer2.json:3]:
```suggestion
    "description": "Audio player which supports music collection playback, lyrics display, format conversion and many other functions.",
```

Grammar

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/musicplayer2.json (hunk 4 above)
Comment (hu3rror) [reply bucket/musicplayer2.json:3]: OK, I directly copied the English description of the project. It looks like he's using machine translation to translate the text.

**Pull request merged**